roles/lib_openshift/src/class/oc_route.py

```diff
@@ -55,7 +55,13 @@ def create(self):
 
     def update(self):
         '''update the object'''
-        # need to update the tls information and the service name
+        # when the host attribute is being updated, we need to delete and recreate
```

Comment:
@detiber, this fails as well.

`[root@ip-172-31-54-78 ~]# oc replace -f droute`
`The Route "docker-registry" is invalid: spec.host: Invalid value: "registry1.<removed>.openshift.com": field is immutable`

The bugzilla was updated on why we have to delete/recreate.